packages/documentation/src/pages/forms/available-features-and-usage-guidelines.mdx

```diff
@@ -438,12 +440,16 @@ preSubmitInfo: {
   field: 'privacyAgreementAccepted',
   label: <span>I have read and accept the <a href="/privacy">privacy policy</a>.</span>,
   error: 'You must accept the privacy policy before continuing',
+  customComponent: ImportedCustomComponent
 }
 ```
-
+#### default PrivacyAgreementTemplate:
 ![Required checkbox before form submission](../../images/forms/Required-Checkbox.jpg)
 
-### Usage guidelines
+### example of customComponent override:
+![Required checkbox before form submission](../../images/forms/preSubmitInfo-customComponent.jpeg)
+
+#### Usage guidelines
```

Comment:
Does the `customComponent` need to do anything in particular?

For example, if I wanted my `customComponent` to require the user to select a certain item from a `<select>`, what do I need to know to make that happen?

Reply:
Good catch, adopted all your title suggestions, will be more descriptive about how to handle the select with a customComponent

Reply:
Updated naming and added description